### PR TITLE
Launch multitask job with parameters

### PR DESCRIPTION
**Pull request by Vakha**
## Proposed changes

Since multi-task job were introduced, dbx wasn't updated to support parameters override for such jobs, as they have different structure. 
This PR adds support for multi-task job structures for `launch` and `launch` with `--as-run-submit`.

https://github.com/databrickslabs/dbx/issues/120

## Types of changes

What types of changes does your code introduce to dbx?
_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

Would be nice to have an option to not only override but also attach parameters.

### CODE REVIEW

File: dbx/commands/launch.py [hunk 1]
```diff
@@ -327,6 +334,16 @@ def launch(self) -> Tuple[Dict[Any, Any], Optional[str]]:
         job_data = matching_jobs[0]
         job_id = job_data["job_id"]
 
+        is_multi_task = False
+        try:
+            is_multi_task = job_data["settings"]["format"] == "MULTI_TASK"
+        except KeyError:
+            pass
```
Comment (renardeinside) [inline dbx/commands/launch.py:341]: could you please make it a bit simpler, for instance, this one-liner?

```
is_multi_task = job_data.get("settings", {}).get("format") == "MULTI_TASK"
```
?

Comment (Vakha) [reply dbx/commands/launch.py:341]: Nice! Will do 👍

**Comment by renardeinside** [comment]
hi @Vakha , 

thanks a lot for contributing to dbx. 

Just to verify - have you checked [the contributing](https://github.com/databrickslabs/dbx/blob/main/CONTRIBUTING.md) and sent the CLA to cla@databricks.com?

**Comment by Vakha** [comment]
> hi @Vakha ,
> 
> thanks a lot for contributing to dbx.
> 
> Just to verify - have you checked [the contributing doc](https://github.com/databrickslabs/dbx/blob/main/CONTRIBUTING.md) and sent the CLA to [cla@databricks.com](mailto:cla@databricks.com)?

I saw it but I don't really get how exactly I have to sign it?

**Comment by Vakha** [comment]
@renardeinside I've sent it to `cla@databricks.com` but I got failure from Mail Delivery Subsystem saying that this email doesn't exist. Are you sure it is correct one?

**Comment by renardeinside** [comment]
hey @Vakha , 
process is also a bit new to me - let me get some information internally and I'll get back to you soon

**Comment by renardeinside** [comment]
hi @Vakha , 
the CLA process wasn't well established, but this time it should be better :) 
Please follow the instructions in the contrib/CONTRIBUTING.md on how to fill in, sign, and whom to send the CLA.

**Comment by Vakha** [comment]
Hi @renardeinside,
I've tried to send it to legal-product@databricks.com and ivan-trusov@databricks.com as CC, but it failed again with "Mail delivery error" 

```
Address not found
--
Your message wasn't delivered to ivan-trusov@databricks.com because the address couldn't be found, or is unable to receive mail.
```

**Comment by renardeinside** [comment]
hi @Vakha , 
Could you please send it once again it to legal-product@databricks.com + cc: ivan.trusov@databricks.com (dot instead of the dash).

**Comment by Vakha** [comment]
@renardeinside, done ✅

**Comment by renardeinside** [comment]
hi @Vakha , 
sorry for the communication break, we've got some other priorities. 
I've implemented multitask jobs support in a separate PR, it would be take too much time to merge the branches with all the conflicts I've introduced. 

Thanks a lot for raising the issue, it will be released as a part of 0.6.6

**Pull request closed without merging**